server/tests/controller/user/signUp.spec.js

```diff
@@ -0,0 +1,63 @@
+import chai from "chai";
+import uniqid from "uniqid";
+import chaiHttp from "chai-http";
+import app from "../../../index";
+import models from "../../../models";
+import userData from "../../fixtures/models/userData";
+import { generateJWT, decodeJWT, getJWTConfigs } from "../../../helpers/jwt";
+
+let token;
+chai.use(chaiHttp);
+const { expect, assert } = chai;
```

Comment:
can you do this from line 1. something like import chai, {expect, assert} from chai